src/Validator/PostCode.php

```diff
@@ -140,7 +140,7 @@ class PostCode extends AbstractValidator
         'MY' => '\d{5}',
         'MV' => '\d{5}',
         'MT' => '[A-Z]{3}[ ]?\d{2,4}',
-        'MU' => '(\d{3}[A-Z]{2}\d{3})?',
+        'MU' => '\d{5}',
```

Comment:
Is this correct? You mentioned in the description that it changed to a plain **six**-digit number, but the regex here is for **five**…